test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -364,14 +364,21 @@ public void testHasColumn() throws Exception {
         GsaSpecialColumns.GSA_TIMESTAMP));
   }
 
+  private Acl buildAcl(ResultSet rs, String delim, String namespace)
```

Comment:
I would have left this alone with a TODO, since we're planning to replace it entirely, or you could have used executeQueryAndNext instead of executeQuery. Just mentioning other ideas for posterity, since we're planning to sweep this away anyway.

Reply:
Agree. Keeping as is for now.